client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -78,71 +227,267 @@ const EditProfilePage = () => {
     fileInputRef.current?.click();
   };
 
+  const getDisplayName = () => {
+    return profile?.nickName || profile?.firstName;
+  };
+
   if (status === "loading" || !userID) {
     return <div>Loading...</div>;
   }
 
   return (
-    <div className="bg-gray-900 flex flex-col items-center justify-start min-h-screen p-4 pt-20 text-white">
-      <h1 className="font-extrabold mb-4 mt-16 text-4xl">Edit Profile</h1>
-      <div className="bg-gradient-to-r flex flex-col from-gray-700 items-center max-w-4xl pb-6 pl-6 pr-6 pt-6 rounded-lg shadow-lg to-gray-900 via-gray-800 w-full">
-        <form onSubmit={handleSubmit} className=" w-full">
+    <div className='bg-gray-900 flex flex-col items-center justify-start min-h-screen p-4 pt-20 text-white'>
+      <h1 className='font-extrabold mb-4 mt-16 text-4xl'>
+        Hello {getDisplayName()}, Edit Your Profile Here
+      </h1>
+      <div className='bg-gradient-to-r flex flex-col from-gray-700 items-center max-w-4xl pb-6 pl-6 pr-6 pt-6 rounded-lg shadow-lg to-gray-900 via-gray-800 w-full'>
+        <form onSubmit={handleSubmit} className=' w-full'>
```

Comment:
The input fields in the form are repetitive. You can create a reusable component and expose a few props for the few things that are different. 

`<CustomInput label={...} onChange={()=>{}} />`

Reply:
@JimmyTran48 This would be a great refactor. But I would be ok moving it to a subsequent ticket. @Mnelson98 's call.

Reply:
True that. Done!